### ISSUE TITLE
Privacy manifest file for iOS is missing

### ISSUE THREAD

**Issue by oremlawi** [Baseflow/flutter_cache_manager#451]
## 🚀 Feature Requests

Kindly consider adding a privacy manifest to your library following the new apple policy by adding `PrivacyInfo.xcprivacy` file to your project.

### Contextualize the feature
For more information, please check [Apple's documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).

### Platforms affected (mark all that apply)
- [ ] :iphone: iOS

**Comment by nnk-ymmt** [comment]
Hello @mvanbeusekom

flutter_cache_manager seems to be using `sqflite`.
It's already added `PrivacyInfo.xcprivacy` at [v2.3.1](https://github.com/tekartik/sqflite/blob/v2.3.1/sqflite/ios/Resources/PrivacyInfo.xcprivacy).
Please upgrade the version.

Thanks.

**Comment by mvanbeusekom** [comment]
Hi @nnk-ymmt,

The `flutter_cache_manager` uses the following constraint for the `sqflite` plugin: `^2.0.0+4`. 

Notice the `^`, this means all minor and patch updates should automatically be resolved. Meaning if you run a `flutter pub upgrade` the latest version of `sqflite` (version 2.3.3 at the time) will used. You can check the actual version being used in the `pubspec.lock` file of your application.

In other words this should not be a blocker. Otherwise the `flutter_cache_manager` doesn't contain any native iOS code and should not need a `PrivacyInfo.xcprivacy` file.

Just for everybody's sanity I have created a PR updating all dependencies: https://github.com/Baseflow/flutter_cache_manager/pull/453
